aquadoggo/src/db/stores/document.rs

```diff
@@ -519,60 +519,59 @@ mod tests {
     use p2panda_rs::test_utils::fixtures::{
         operation, random_document_id, random_document_view_id, random_operation_id,
     };
+    use p2panda_rs::test_utils::memory_store::helpers::{populate_store, PopulateStoreConfig};
     use p2panda_rs::WithId;
     use rstest::rstest;
 
     use crate::db::stores::document::DocumentView;
     use crate::test_utils::{
-        build_document, doggo_schema, test_db, TestDatabase, TestDatabaseRunner,
+        build_document, populate_and_materialize, populate_store_config, test_runner, TestNode,
     };
 
     #[rstest]
     fn insert_and_get_one_document_view(
-        #[from(test_db)]
+        #[from(populate_store_config)]
         #[with(2, 1, 1)]
-        runner: TestDatabaseRunner,
+        config: PopulateStoreConfig,
```

Comment:
Naming-wise I stumble a little bit here thinking:

"I populate the `PopulateStoreConfig` with `populate_store_config`" .. but it would feel more natural to say: "I populate the `TestStoreConfig` with `populate_store_config` or something similar?

Reply:
Ah, now I understand the pattern

Reply:
Forgot about how `rstest` fixtures work :+1: